src/app/components/primary-navbar/primary-navbar.component.html

```diff
@@ -1,10 +1,12 @@
 <ng-template #desktop>
     <div class="desktop-navbar" *ngIf="showNav">
         <div class="nav-wrapper">
-            <div class="logo" [routerLink]="['/home']">
-                <span class="logo-image"></span>
-                <span id="clark-logo" class="logo-link">CLARK</span>
-            </div>
+            <a [routerLink]="['/home']" target="_blank" style="text-decoration: none;">    
```

Comment:
I'd also suggest moving the in-line styling to the style sheet, but that's a personal preference.

Reply:
Good tip! I'll do that